helix-core/src/test/java/org/apache/helix/integration/task/TestQuotaBasedScheduling.java

```diff
@@ -116,6 +117,11 @@ public void beforeClass() throws Exception {
     _jobCommandMap = Maps.newHashMap();
   }
 
+  @AfterClass
+  public void afterClass() throws Exception {
```

Comment:
I think it is not necessary. The super afterClass will still be called if there is no class with the same name override it in the child class. Please have a try.

Reply:
removed.